hat/test/org/hatdex/hat/api/controllers/ContractDataSpec.scala

```diff
@@ -25,18 +25,15 @@
 package org.hatdex.hat.api.controllers
 
 import org.hatdex.hat.api.HATTestContext
-import play.api.Logger
 import play.api.libs.json.{ JsValue, Json }
 import play.api.test.Helpers._
 import play.api.test.{ FakeRequest, Helpers }
 
 import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
 
 class ContractDataSpec extends ContractDataContext {
-  import scala.concurrent.ExecutionContext.Implicits.global
-
-  val logger: Logger = Logger(this.getClass)
```

Comment:
Just a random question. Is the logger needed in tests? Can it be useful in certain scenarios?

Reply:
as you can see I removed it 😉

Reply:
the only time a logger might be used in a test is to do old school debugging, but this is clearly not something desirable, and it should not be copy / pasted in every test class !